### PR TITLE
Day 2. Homework

### PR DESCRIPTION
**Pull request by eclimov**
Entities created. 
+ minor security improvements

### CODE REVIEW

File: src/Entity/Affiliate.php [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+     * @var bool
+     * @ORM\Column(type="boolean")
+     */
+    private $active;
+
+    /**
+     * @var datetime
+     * @ORM\Column(type="datetime")
+     */
+    private $created_at;
```
Comment (gregurco) [inline src/Entity/Affiliate.php]: camelCase

File: src/Entity/Affiliate.php [hunk 2]
```diff
@@ -0,0 +1,155 @@
+<?php
+namespace App\Entity;
+
+use DateTime;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * Class Affiliate
```
Comment (gregurco) [inline src/Entity/Affiliate.php]: remove this useless comment `Class Affiliate`

File: src/Entity/Job.php [hunk 3]
```diff
@@ -0,0 +1,408 @@
...
+
+    /**
+     * @var string
+     * @ORM\Column(type="string", length=255)
+     */
+    private $company;
+
+    /**
+     * @var string
+     * @ORM\Column(type="string", length=255)
```
Comment (gregurco) [inline src/Entity/Job.php]: logo and url are nullable

File: src/Entity/Job.php [hunk 4]
```diff
@@ -0,0 +1,408 @@
...
+     * @var string
+     * @ORM\Column(type="text")
+     */
+    private $description;
+
+    /**
+     * @var string
+     * @ORM\Column(type="text")
+     */
+    private $how_to_apply;
```
Comment (gregurco) [inline src/Entity/Job.php]: also camelCase

**Review by gregurco — CHANGES_REQUESTED** [review]

**Comment by eclimov** [comment]
Fixed

**Comment by gregurco** [comment]
@ksardedik :+1:

**Pull request merged**